CHANGELOG.rst

```diff
@@ -2,11 +2,34 @@
 Changelog
 =========
 
-Upgrade notes
--------------
+2.7.10 (2024-11-22)
+===================
+
+Upgrade procedure
+-----------------
+
+* ⚠️ Ensure that the form variables initial values for selectboxes components match with
+  their respective component default values.
+  
+  After upgrading to 2.7.10, run the ``app/bin/fix_selectboxes_component_default_values.py``
+  script. This script automatically fixes all initial values of the existing selectboxes
+  component form variables.
+
+  .. code-block:: bash
+
+       # in the container via ``docker exec`` or ``kubectl exec``:
+       python src/manage.py app/bin/fix_selectboxes_component_default_values.py
+
+Periodic bugfix release
```

Comment:
This one typically goes right after the version number/date heading, so above the upgrade procedure :)